_top-level/support.md

```diff
@@ -11,12 +11,11 @@ Don't hesitate to ask questions if you are having trouble with MyRocks.
 
 ### GitHub issues
 
-The [GitHub issues](https://github.com/facebook/mysql-5.6/issues) page is a good place to ask
-questions, find answers, and report issues.
+Use [GitHub issues](https://github.com/facebook/mysql-5.6/issues) to report bugs, issues and feature requests for the MyRocks codebase.
 
 ### Facebook Group
 
-<https://www.facebook.com/groups/mysqlonrocksdb/>
+Use the [MyRocks Facebook group](https://www.facebook.com/groups/mysqlonrocksdb/) for general questions and discussion about MyRocks.
```

Comment:
Actually, we want the mailing list as the first stop shop for questions and discussions. The FB group should be the fallback.